onboarding/array.py

```diff
@@ -9,7 +9,14 @@ def find_min_index(arr):
     '''
 
     # insert code here
-    pass
+    min = arr[0]
+    minpos = 0
```

Comment:
Super super nitpicky, but general convention is snakecase so for legibility I would always encourage `min_pos` 😀